pkg/nodebootstrap/userdata.go

```diff
@@ -100,7 +100,7 @@ func makeKubeletConfigYAML(spec *api.ClusterConfig, ng *api.NodeGroup) ([]byte,
 
 	// use a map here, as using struct will require us to add defaulting etc,
 	// and we only need to add a few top-level fields
-	obj := map[string]interface{}{}
+	obj := api.InlineDocument{}
```

Comment:
Should the comment be updated?

Reply:
out of curiosity why was this changed?

Reply:
`api.inlineDocument` is a typedef of `map[string]interface{}` and is used throughout the source. (e.g., i saw it in `userdata_bottlerocket.go`).

i changed this line too because in the code i added below i use it instead of the generic `map[string]interface{}`, to keep it consistent in this file